srcpkgs/tessen/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'tessen'
+pkgname=tessen
+version=2.2.1
+revision=1
+build_style=gnu-makefile
+hostmakedepends="scdoc"
+checkdepends="shellcheck shfmt"
+short_desc="Interactive menu to autotype and copy pass and gopass data"
+maintainer="reedts <j.reedts@gmail.com>"
+license="GPL-2.0-only"
+homepage="https://git.sr.ht/~ayushnix/tessen"
+distfiles="https://git.sr.ht/~ayushnix/tessen/refs/download/v${version}/tessen-${version}.tar.gz"
+checksum=31d510d5ffac5825bc7e63f13283c5d432eb102db2d2483ee937c9135c8cfc04
+
+post_install() {
+	vcompletion completion/tessen.bash-completion bash
```

Comment:
[from what I see in the makefile](https://git.sr.ht/~ayushnix/tessen/tree/v2.2.1/item/Makefile#L32), this should be done by `make install`

Reply:
The new commit should be as minimal as it gets